packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js

```diff
@@ -42,9 +72,10 @@ NativeValue.propTypes = {
 		PropTypes.string,
 	]),
 	edit: PropTypes.bool,
-	onClick: PropTypes.func,
+	onSelect: PropTypes.func,
```

Comment:
looking at the implementation this function is required and so should be tagged as required here